### DISCUSSION TITLE
커스텀 월드 개발시 사용자 언어 설정에 관한 질의드립니다.

### DISCUSSION THREAD

**Discussion by hina9** [naverz/zepeto-studio-kor#702] [5. Q&A]
유니티 상으로 커스텀 월드 개발을 하고, 제패토 업로드 시

사용자 언어나,  설정등은 커스텀이 가능한가요?

혹시 불가능 하다면 사용자가 외국인이었을경우 표기되는 부분은 어떻게 진행되는지요? 확인 요청드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, UnityEngine.Applicaiton.systemLanguage 사용이 가능합니다.
참고로 제페토 앱 언어 설정은 디바이스의 언어 설정을 따라갑니다. 
감사합니다.
